From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sat, 22 Apr 2017 21:27:04 +0200
Subject: ramips: sysupgrade: move nand_do_upgrade call to platform_do_upgrade

All targets with NAND support should gradually move their nand_do_upgrade
calls from platform_pre_upgrade to platform_do_upgrade.

Signed-off-by: Matthias Schiffer <mschiffer@universe-factory.net>

diff --git a/target/linux/ramips/base-files/lib/upgrade/platform.sh b/target/linux/ramips/base-files/lib/upgrade/platform.sh
index 7f5b1dd070b1bfe0b86b093a55165b5a139eb37c..a5cca2080d3b96cef92286b1b15f4f8832d9f8c5 100755
--- a/target/linux/ramips/base-files/lib/upgrade/platform.sh
+++ b/target/linux/ramips/base-files/lib/upgrade/platform.sh
@@ -254,20 +254,13 @@ platform_nand_pre_upgrade() {
 	esac
 }
 
-platform_pre_upgrade() {
+platform_do_upgrade() {
 	local board=$(ramips_board_name)
 
 	case "$board" in
     	ubnt-erx)
 		nand_do_upgrade "$ARGV"
 		;;
-	esac
-}
-
-platform_do_upgrade() {
-	local board=$(ramips_board_name)
-
-	case "$board" in
 	*)
 		default_do_upgrade "$ARGV"
 		;;
